fecgov/fec-pattern-library
# Feature/206 upgrade node

## Summary of changes

- Addresses #206 
_Upgrades to Node 18.13 NPM 8.19.3_


## Considerations
Upgrading above Fractal 1.4.1 and mandelbrot 1.5 would require some pretty big code re-writes. With this PR, we have upgraded those packages from where we were running them before, but we have to pin them at these versions. There's a current ticket in [for a big pattern library audit,](http://localhost:3000  ) which I think should happen and then do more Fractal upgrades. 


## Test
- pull this branch (`gh pr checkout 208`)
- pyenv virtualenv 3.9.15 pattern
- pyenv activate pattern
- make sure you're using the correct node versions (`nvm install 18.13.0') 
- npm upgrade
- npm i
- npm run build
- npm run start
